client/ayon_core/tools/common_models/projects.py

```diff
@@ -1,17 +1,24 @@
 import contextlib
-from abc import ABCMeta, abstractmethod
+from abc import ABC, abstractmethod
+from typing import Literal, Dict, Any
 
 import ayon_api
-import six
 
 from ayon_core.style import get_default_entity_icon_color
 from ayon_core.lib import CacheItem, NestedCacheItem
 
 PROJECTS_MODEL_SENDER = "projects.model"
+StatusStatesType = Literal["not_started", "in_progress", "done", "blocked"]
```

Comment:
It's good to note that adding `Literal` typing makes this support Py3.8+ only. [It's added in Py3.8](https://docs.python.org/3/library/typing.html#typing.Literal). Is that a problem? 

I believe Maya 2022 is Py3.7 still maybe?

Reply:
Hmm, might be.

Reply:
Changed to use `str`.